deepmd/pt/model/descriptor/se_atten.py

```
@@ -586,6 +631,145 @@ def forward(
+        extended_coord: torch.Tensor,
+        extended_atype: torch.Tensor,
+        extended_atype_embd: Optional[torch.Tensor] = None,
+        mapping: Optional[torch.Tensor] = None,
+        natoms: Optional[torch.Tensor] = None,
+    ):
+        del mapping, natoms
+        assert extended_atype_embd is not None
+        nframes, nloc, nnei = nlist.shape
+        atype = extended_atype[:, :nloc]
+        nb = nframes
+        nall = extended_coord.view(nb, -1, 3).shape[1]
```

Ruff

Local variable `nall` is assigned to but never used

Remove assignment to unused variable `nall`

(F841)